apps/masterbots.ai/components/routes/thread/user-thread-panel.tsx

```
@@ -165,6 +172,28 @@ export default function UserThreadPanel({
+
+    if (thread) {
+      setActiveThread(thread)
+      setThreads([thread])
+      setIsContinuousThread(true)
+      setTotalThreads(1)
+      setCount(1)
+      setIsOpenPopup(true)
+    }
+  }
+
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: getThreadByContinuousThreadId

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)